ThingsBoard/README.md

```diff
@@ -0,0 +1,120 @@
+After installation, you can check if it is installed correctly by typing the following commands in your terminal:
+
+* Linux
+```bash
+python3 --version
+pip3 --version
+```
+If python3 is not recognized, try "python"
+
+* Windows:
+```bash
+python --version
+pip --version
+```
+
+## Installation
+
+### 1. Create virtual environment:
+First, you need to create virtual environment (either with python venv module or anaconda-python):
+
+Note, that **"ENV_NAME"** is the name of the environment you’re creating.
+
+Enter main directory of the project (iot-starter):
+```bash
+cd iot-starter
+```
+
+* For anaconda (either in terminal - Linux or anaconda prompt - Windows):
+  ```
+  conda create --name ENV_NAME python=3.7 pip
+  ```
+
+* For venv:
+  * Linux:
+  ```
+  python3 -m venv ENV_NAME
+  ```
+
+  * Windows:
+  ```
+  python -m venv ENV_NAME
+  ```
+
+Next, you should activate it. This step is platform dependent:
+#### Windows
+* For anaconda:
+  ```
+  conda activate ENV_NAME
+  ```
+
+* For venv:
+  ```
+  ENV_NAME/Scripts/activate.bat
+  ```
+
+#### Linux/ Mac OS
+* For anaconda:
+  ```
+  conda activate ENV_NAME
+  ```
+
+* For venv:
+  ```
+  source ENV_NAME/bin/activate
+  ```
+
+### 2. Install requirements
+
+After you've created virtual environment, your current directory should be "iot-starter"
+
+``` 
+pip install pyserial cryptography click future pyelftools setuptools
+pip install -r ThingsBoard/requirements.txt
+```
+
+## Set up local ThingsBoard server
+
+### 1. Installation
+
+## Linux
+To create your own ThingsBoard server on Linux (Ubuntu), just follow this [instruction](https://thingsboard.io/docs/user-guide/install/ubuntu/).
+
+## Windows
+For Windows users, instruction can be found [here](https://thingsboard.io/docs/user-guide/install/windows/).
+
+### 2. Device profile set up
+
+Once you finally have your ThingsBoard ready, open your browser and navigate to server webpage. Default link is:
+```
+http://localhost:8080/
+```
+Now go ahead and log in to your server. If you have specified *-loadDemo* during execution of the installation script, the default credentials are:
+* System Administrator: sysadmin@thingsboard.org / sysadmin
+* Tenant Administrator: tenant@thingsboard.org / tenant
+* Customer User: customer@thingsboard.org / customer
+
+You can always change passwords for each account in account profile page.
+
+Now the only thing you need to do is obtain two keys needed for device registering. To do this, click on the **Device profiles** tab on the left and hit the profile name you want to use (you can use default one or create a new profile). Toggle on edit mode, then go to **Device provisioning** tab and change the **Provision strategy** to 'Allow to create new devices'. Apply changes and now two new fields have appeared - **Provision device key** and **Provision device secret**. Please copy them and save for later.
```

Comment:
1. (...) you need to do is **to** (...)
2.  (you can use **the** default ~~one~~ or create a new ~~profile~~ one
3.  (...) then go to **the** **Device provisioning** (...)
4.  Apply changes and now two new fields have appeared -> After applying changes, two new fields should appear